api/v1alpha3/awscluster_types.go

```diff
@@ -89,6 +89,11 @@ type Bastion struct {
 	// with a public ip to access the VPC private network.
 	// +optional
 	Enabled bool `json:"enabled"`
+
+	// CIDRBlocks is a list of CIDR blocks allowed to access the bastion host.
+	// They are set as ingress rules for the Bastion host's Security Group (defaults to 0.0.0.0/0).
+	// +optional
+	CIDRBlocks []string `json:"cidrBlocks"`
```

Comment:
This will need to be `omitempty` and we will not be able to distinguish between nil and [] (SIG API Machinery advises against trying to distinguish). We'll likely want to use a separate bool field to disable ingress.

Reply:
ok, so if the value is empty we default to `0.0.0.0/0` and will use a boolean as an option to have to rules.

Reply:
@ncdc Added a new boolean field called `NoIngressRules`.  I couldn't think of a better name. 😀

Let me know if you want something different. Also, I wasn't sure if we want `omitempty` for the boolean. So just want to confirm that.